### PR TITLE
Fix and Refactor code UI calendar index

### PR DESCRIPTION
**Pull request by tqtrung09**

### CODE REVIEW

**Comment by daodat18plus** [comment]
OK

File: app/assets/javascripts/calendar.js [hunk 1]
```diff
@@ -203,6 +214,82 @@ $(document).on('page:change', function() {
...
+  });
+
+  $('.clstMenu-child').click(function() {
+    var windowH = $(window).height();
+    var position = $(this).offset();
+    $('#id-of-calendar').html($(this).attr('id'));
+    var menu_height = $('#menu-of-calendar').height();
+    if ((position.top + 12 + menu_height) >= windowH ) {
+      $('#menu-of-calendar').
+      css({'top': position.top - menu_height - 2, 'left': position.left});
```
Comment (vietbq) [inline app/assets/javascripts/calendar.js]: convention

File: app/assets/javascripts/calendar.js [hunk 2]
```diff
@@ -203,6 +214,82 @@ $(document).on('page:change', function() {
...
+    }else {
+      $('#menu-of-calendar').
+      css({'top': position.top + 12, 'left': position.left});
```
Comment (vietbq) [inline app/assets/javascripts/calendar.js]: convention

**Comment by vietbq** [comment]
Ok

**Pull request merged**